src/data/javascript-quiz.ts

```diff
@@ -1369,6 +1379,16 @@ const javascriptQuiz = [
       "The event loop is responsible for executing the code, processing events, and monitoring the callback queue and stack. ",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/EventLoop",
   },
+  {
+    Question: "Which ES6 collection stores unique values?",
+    Answer: "Set",
+    Distractor1: "Maps",
+    Distractor2: "Object",
+    Distractor3: "Array",
+    Explanation:
+      "Set is ES6 Collection which lets you store unique values of any type. e.g. const food = new Set(['Bread', 'Butter', 'Bread', 'Eggs']); food = {'Bread', 'Butter', 'Eggs'}. Set discards the repeating 'Bread'.",
```

Comment:
Let's shorten this a little bit.

```suggestion
      "A JavaScript Set is a type of data structure that stores unique values. ",
```